src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -94,12 +88,6 @@ if not [%NET46_PUBLISH_ERROR_LEVEL%]==[0] (
   exit /b %NET46_PUBLISH_ERROR_LEVEL%
 )
 
-:: Copy some roslyn files which are published into runtimes\any\native to the root
-Robocopy "%TOOLRUNTIME_DIR%\runtimes\any\native" "%TOOLRUNTIME_DIR%\."
-
-:: Microsoft.Build.Runtime dependency is causing the MSBuild.runtimeconfig.json buildtools copy to be overwritten - re-copy the buildtools version.
-Robocopy "%BUILDTOOLS_PACKAGE_DIR%\." "%TOOLRUNTIME_DIR%\." "MSBuild.runtimeconfig.json"
-
 :: Copy Portable Targets Over to ToolRuntime
 if not exist "%PACKAGES_DIR%\generated" mkdir "%PACKAGES_DIR%\generated"
```

Comment:
We don't, I kept this for the microbuild stuff, but if that is not required I'll go ahead and nuke this as well